src/pages/register/Register.js

```diff
@@ -1,7 +1,8 @@
 import React, { useState } from "react";
+import Navbar from "../navbar/Navbar.js";
 import "./Register.css";
 
-function Register() {
+function Register({ isLoggedIn }) {
```

Comment:
I don't think we will need to pass isLoggedIn as a prop to any component except the navbar. 

What we will pass as props to login and register components is a handler function that creates a new user or logs a user in. These will be defined in App.js. 

I think this is how it will work: When a user clicks on submit in register or login, the handler will be called and user info from the forms will be sent as an argument to the handler function. The handler function will use a fetch call to send the user info to the BE and either find or create the user in the DB. Then the BE will send back the user info, which we will then save to state in App.js. Authenticaton info will also be sent back from the BE, but I'm very foggy on how all of that works! I'll need to do a bit of research and looking at past projects. We can definitely ask Frank for some help as well.

Reply:
UPDATE: sorry Alejandro, I've been rethinking passing the handler down from registration and login - I think we will not do that. Check out PRAC-67. I think it's a cleaner approach!